### PR TITLE
[Swegon] Support for RAW PDUs

### PR DESCRIPTION
**Pull request by paulianttila**
Added support for RAW PDU which allows to use any serial port UDP converters.

Signed-off-by: Pauli Anttila <pauli.anttila@gmail.com>

### CODE REVIEW

File: bundles/binding/org.openhab.binding.swegonventilation/src/main/java/org/openhab/binding/swegonventilation/internal/SwegonVentilationBinding.java [hunk 1]
```diff
@@ -168,22 +168,28 @@ private State convertDeviceValueToOpenHabState(Class<? extends Item> itemType, I
...
 
         public void setInterrupted(boolean interrupted) {
             this.interrupted = interrupted;
             this.interrupt();
+            try {
+                if (connector != null) {
+                    connector.disconnect();
+                }
+            } catch (SwegonVentilationException e) {
+                logger.error("Error occured when closing connection", e);
```
Comment (9037568) [inline bundles/binding/org.openhab.binding.swegonventilation/src/main/java/org/openhab/binding/swegonventilation/internal/SwegonVentilationBinding.java]: This should be a WARN or lower.
spelling fix: "occurred"

File: bundles/binding/org.openhab.binding.swegonventilation/src/main/java/org/openhab/binding/swegonventilation/internal/SwegonVentilationBinding.java [hunk 2]
```diff
@@ -248,7 +254,7 @@ public void run() {
 
                 } catch (SwegonVentilationException e) {
 
-                    logger.error("Error occured when received data from Swegon ventilation system", e);
+                    logger.debug("Error occured when received data from Swegon ventilation system", e);
```
Comment (9037568) [inline bundles/binding/org.openhab.binding.swegonventilation/src/main/java/org/openhab/binding/swegonventilation/internal/SwegonVentilationBinding.java]: spelling fix: "occurred"

**Comment by paulianttila** [comment]
Fixes done

**Comment by 9037568** [comment]
Thanks!

**Pull request merged**